### PR TITLE
{bio}[foss-2020b] pdbfixer-1.7 required for alphahelix created

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Requires OpenMM from PR #[13452](https://github.com/easybuilders/easybuild-easyconfigs/pull/13452)

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s/'],
+}
+
+# sanity_check_commands = ['%(namelower)s-count --help']
+
+# sanity_pip_check = True
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:31]: this should be enabled

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+# options = {'modulename': '%(name)s'}
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:33]: Remove

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s/'],
+}
+
+# sanity_check_commands = ['%(namelower)s-count --help']
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:29]: Is this available? if so, uncomment, else just remove

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:31]: Done. That helped as a dependency was missing.

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:33]: Done. That was from a template file I used.

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb (hunk 3 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:29]: Not as such but I changed it to
`sanity_check_commands = ['%(namelower)s --help']`
which worked. Thanks.

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'pdbfixer'
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:3]:
```suggestion
name = 'PDBFixer'
```

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb (hunk 4 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb:3]: Done. I think I slowly understand your nomenclature. I hope.

File: easybuild/easyconfigs/p/PDBfixer/PDBfixer-1.7-foss-2020b.eb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    ('OpenMM', '7.5.1'),
+    ('h5py', '3.1.0'),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s/'],
```
Comment (Micket) [inline easybuild/easyconfigs/p/PDBfixer/PDBfixer-1.7-foss-2020b.eb:27]:
```suggestion
    'files': ['bin/%(namelower)s'],
    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s/'],
```

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PDBfixer/PDBfixer-1.7-foss-2020b.eb (hunk 5 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/p/PDBfixer/PDBfixer-1.7-foss-2020b.eb]: Done. Thanks for pointing that out.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/34f855d367749a01c015e0f6ec7183d3 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@sassy-crick I overlooked this, and already included an easyconfig file for PDBFixer in #13744...

**Comment by sassy-crick** [comment]
@boegel That happens, specially with the speed the project was developing and the number of dependencies it has. 
Shall I close this PR in that case?

**Comment by boegel** [comment]
@sassy-crick Yes, I think closing this is the best option.

**Pull request closed without merging**